joomla/joomla-cms
# [4.4] Fix wrong redirection for saving user account

Pull Request for Issue #40573.

### Summary of Changes
This PR fix a regression from PR https://github.com/joomla/joomla-cms/pull/39038 . As of right now, if someone does not have `core.admin` (Configure ACL & Options) permission edit a user account and save it, he will always be redirected to admin dashboard instead of staying on edit screen (for Save action) or stay on Users Management screen (for Save & Close action)


### Testing Instructions
- Login to administrator area of your site using an account does not have **Configure ACL & Options** permission in com_users. Please make sure you give him core.manage (Access Administration Interface) permission and other permissions so that he can access to Users Management screen to edit user account. 
- Edit a user account, save it.

### Actual result BEFORE applying this Pull Request

- You are being redirected to administrator Home Dashboard

### Expected result AFTER applying this Pull Request
- You stay on user edit screen or users management screen (depend on you use Save or Save & Close button)
- Additional, please test and make sure the issue https://github.com/joomla/joomla-cms/issues/39028 is still fixed with this PR, too

### Link to documentations
